### DISCUSSION TITLE
For you page (Beta)

### DISCUSSION THREAD

**Discussion by angeloboustany** [community/community#13445] [Other Feature Feedback, Questions, & Ideas]
Hello, I was checking out the For you page, which is said to be in Beta mode, and I have a suggestion.
I feel like the text ("Recommended for you") above each section is redundant like we already know that the page is a For You and the content there is recommended.
Anyways I love that new feature, keep it up!

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
